src/renderers/react/ElementWrapper.tsx

```diff
@@ -37,10 +38,25 @@ const Body = styled("div")`
 
 const Panel = styled("div")<{ isSelected: boolean }>`
   background-color: ${({ isSelected }) =>
-    isSelected ? "#b3d7fe" : neutral[97]};
+    isSelected ? "#b3d7fe !important" : neutral[97]};
```

Comment:
Haha. It works – but it's brutal!

I'm sure you've tried lots of things – is there any way we can avoid this? Appreciate we don't necessarily want to be passing selection state around.

My first thought is a transparent coloured overlay and some conditional styling for selection, but you may have already been down this road! 

We could also use the [component selector](https://styled-components.com/docs/advanced#referring-to-other-components) pattern for CSS precedence.

Not a blocker, but `!important` does make me itch a bit 😅

Reply:
Switched to using an overlay as per the suggestion. In my opinion it looks much better.